Classes/common/HTTP/CDTURLSession.m

```diff
@@ -40,6 +42,7 @@ - (instancetype)initWithDelegate:(id<NSURLSessionDelegate>)delegate callbackThre
         _thread = thread;
         
         NSURLSessionConfiguration *config = [NSURLSessionConfiguration defaultSessionConfiguration];
+        _requestInterceptors = @[];
```

Comment:
So I don't understand why you are not assigning the `requestInterceptors` value passed into the init to the method to the backing variable.